docs/application/tizen-studio-0/setup/windows.md

```diff
@@ -0,0 +1,172 @@
+# Install Tizen Studio on Windows
+
+This page explains the process to download and install Tizen Studio on your development hardware.
+
+To have optimal installation experience, follow these steps chronologically:
+
+- System prerequisites
+  - Hardware requirements
+  - Software requirements
+- Setup OpenJDK in Windows
+- Launch installer
+- Verify installation
+
+<style type="text/css">
+a.clickable   { width: 100%; height: 100%; }
+a.clickable:hover   { background-color: ; color: #FFFFF; }
+</style>
+
+## System Prerequisites
+
+Ensure that the following system prerequisites are met:
+
+### Hardware Requirements
+<table>
+  <tr>
+      <td width=150px>CPU</td>
+    <td colspan="3" width=50px>Dual Core, 2Ghz or faster</td>
+  </tr>
+  <tr>
+    <td>Architecture</td>
+    <td width=520px>x32 or x64 </td>
+  </tr>
+  <tr>
+    <td>Memory</td>
+    <td colspan="3">3GB or more </td>
+  </tr>
+  <tr>
+    <td>Disk space</td>
+    <td colspan="3">6 GB or more </td>
+  </tr>
+</table> 
+
+### Software Requirements
+<table>
+  <tr>
+    <th colspan="2" align=left>Windows</th>
+  </tr>
+  <tr>
+    <td width=150px>Version</td>
+    <td width=520px>7/8/10</td>
+  </tr>
+  <tr>
+    <td>Bit</td>
+    <td>32 or 64 bit</td>
+  </tr>
+</table>
+
+<table>
+  <tr>
+    <th>Software</th>
+    <th>Download Link </th>
+  </tr>
+  <tr>
+    <td>Open JDK</td>
+    <td width=520px><a href="https://download.java.net/java/GA/jdk10/10/binaries/openjdk-10_windows-x64_bin.tar.gz" class="clickable" target="_blank">https://download.java.net/java/GA/jdk10/10/binaries/openjdk-10_windows-x64_bin.tar.gz</a></td>
```

Comment:
It's not a good idea to show file URL here. Just provide a URL of  its site.

Reply:
I have made the necessary changes, please review and give your valuable comments.